tests/release/README.md

```diff
@@ -5,98 +5,113 @@ Contains E2E tests related to [Release Service](https://github.com/redhat-appstu
-       - Validate that the SBOM for both components were uploaded to Pyxis stage.
-
-### 5. Ensure ReleasePlan has owner references set (e2e-test-releaseplan-owner-ref.go)
-
-   This test is designed to ensure that the ReleasePlan controller reconciles ReleasePlans to have an owner reference for its application.
-
-      Steps:
-       - Create a ReleasePlan and an Application.
-      Expected Results:
-       - The ReleasePlan has an owner reference for the Application.
-       - If the Application is deleted, the ReleasePlan is also deleted.
-
-### 6. Negative e2e-tests
-We use manual tests to cover negative tests, using filtering labels in Ginkgo framework to enable users to run negative e2e-tests whether all together or each one separately:
-
-**Run all negative release e2e-tests**
-```bash
-./bin/e2e-appstudio --ginkgo.junit-report=report.xml  --ginkgo.focus="release" --ginkgo.label-filter="release-neg" --ginkgo.vv
-```
-**Run one specific negative e2e-test**
-```bash
-e2e-test = negMissingReleaseStrategy || negMissingReleasePlan || negMissingReleasePlanAdmission
-./bin/e2e-appstudio --ginkgo.junit-report=report.xml  --ginkgo.focus="release" --ginkgo.label-filter="<e2e-test>" --ginkgo.vv
-```
-   #### 6.1. A release CR will fail if ReleasePlanAdmission is missing (neg-missing-releaseplanadmission.go).
-
-      Steps:
-       - Create the following CRs: Snapshot, ReleaseStrategy, ReleasePlan, EnterpriseContractPolicy and Release          
-      Expected Results:
-       - We ensure that Release CR fails on Validation and on Release, with a proper message printed out to the user. 
-
-   #### 6.2. A release CR will fail if ReleasePlan is missing (neg-missing-releaseplan.go).
-
-      Steps:
-       - Create the following CRs: Snapshot, ReleaseStrategy, ReleasePlanAdmission, EnterpriseContractPolicy and Release          
-      Expected Results:
-       - We ensure that Release CR fails on Validation and on Release, with a proper message printed out to the user.
-
-   #### 6.3. A release CR will fail if ReleaseStrategy is missing (neg-missing-releasestrategy.go).
-
-      Steps:
-       - Create the following CRs: Snapshot, ReleasePlan, ReleasePlanAdmission, EnterpriseContractPolicy and Release          
-      Expected Results:
-       - We ensure that Release CR fails on Validation and on Release, with a proper message printed out to the user.
+
+### 1. Release service suite 
+
+   The test cases in this suite are to test release-service functionalities.
+   All tests must have the label `release-service`. 
+
+   #### 1.1. The happy path test (happy_path.go)
+
+	This test verifies the functionality of the release-service by creating Application and component CRs to trigger build Pipeline. Once the build PipelineRun succeeds, Snapshot and Release CRs are created by integration-service if releasePlan is configed. Then the Release CR triggers release pipelineRun and the Release CR is got released at last.
+
+      	Steps:
```

Comment:
All of these indented blocks are rendered as code blocks. I know it was like this even before, but I think it's worth fixing.

Reply:
Updated, thanks! Please help to review again.

Reply:
Cool. I wanted to check what the changes in this PR are actually about (other then reorganizing the information), but there is no description - just the title which doesn't say much. Can you provide a bit more detail about what is being updated? Ideally both in the PR description and commit message.